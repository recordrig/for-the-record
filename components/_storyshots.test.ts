import initStoryshots from "@storybook/addon-storyshots";

/**
 * Storyshot's is Storybook's integration with Jest's snapshot testing. It auto-generates
 * snapshots based on stories so that we don't have to manually define them. As Jest iterates
 * through all files matching the `.test` pattern, this initiation function will be executed.
 *
 * See [Jest's documentation on shapshot testing](https://jestjs.io/docs/en/snapshot-testing)
 * and [Storybook's documentation on Storyshot](https://storybook.js.org/docs/testing/structural-testing/).
 */
initStoryshots({});
