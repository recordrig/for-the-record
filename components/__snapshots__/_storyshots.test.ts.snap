// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots TopMenu TopMenu 1`] = `null`;
